components/user-store/org.wso2.carbon.identity.user.store.configuration.ui/src/main/resources/web/userstore_config/index.jsp

```diff
@@ -220,34 +221,34 @@
                         }
 
                 %>
-                <tr id=<%=domainId%>>
+                <tr id=<%=Encode.forHtmlAttribute(domainId)%>>
                     <td style="width: 5%;margin-top:10px;">
                         <input type="checkbox" name="userStores"
-                               value="<%=domainId%>"
+                               value="<%=Encode.forHtmlAttribute(domainId)%>"
                                onclick="resetVars()"
                                class="chkBox"/>
                     </td>
                     <td style="width: 10%;margin-top:10px;">
-                        <a><%=domainId%>
+                        <a><%=Encode.forHtml(domainId)%>
                         </a>
                     </td>
                     <td style="width: 40%;margin-top:10px;">
-                        <a><%=className%>
+                        <a><%=Encode.forHtml(className)%>
                         </a>
                     </td>
                     <td style="width: 45%;margin-top:10px;">
                         <a title="<fmt:message key='edit.userstore'/>"
-                           onclick="edit('<%=domainId%>','<%=className%>');"
+                           onclick="edit('<%=Encode.forJavaScriptAttribute(domainId)%>','<%=Encode.forJavaScriptAttribute(className)%>');"
                            href="#" style="background-image: url(images/edit.gif);" class="icon-link">
                             <fmt:message key='edit.userstore'/></a>
                         <% if (!isDisabled) { %>
                         <a title="<fmt:message key='disable.userstore'/>"
-                           onclick="disable('<%=domainId%>');return false;"
+                           onclick="disable('<%=Encode.forJavaScriptAttribute(domainId)%>');return false;"
```

Comment:
encode domainId URI and then encode for javascript attribute. This is because within the disable function this attribute is used in the query string of a form action